docs/decisions/0007-lti-1p3-decoupling.rst

```diff
@@ -0,0 +1,115 @@
+0007 Decouple LTI 1.3 Launch from XBlock and edX Platform
+#########################################################
+
+Status
+******
+**Draft (=> Accepted)**
+
+Context
+*******
+
+In this document, the terms context and launch information are important. Context describes the technological
+environment or application in which an LTI 1.3 launch occurs. For example, the context may be XBlock, course,
+independently deployable application (IDA), etc. Launch information describes information included in an LTI 1.3 launch.
+Launch information is relative to the context. Launch information may refer to query or form parameters or to claims in
+a JWT.
+
+Currently, LTI 1.3 launches rely on the ``location`` field of the ``LtiConfiguration`` model. The ``location`` field is a
+``UsageKeyField`` and represents the location of the XBlock associated with the ``LtiConfiguration`` instance. This
+association was made because, originally, the XBlock was the source of truth for LTI configuration information and was
+the only context in which LTI launches occurred.
+
+The dependence on the ``location`` field prevents LTI 1.3 launches from being possible outside the context of an XBlock.
+This prevents Open edX from using LTI more widely across the platform and across the ecosystem, because its use is
+restricted to specific contexts in which ``location`` is meaningful.
+
+For example, because ``LTIConfiguration`` instances are associated with a particular ``location``, it is not possible for a
+``LtiConfiguration`` to be reused across different XBlocks (i.e. a course context), even if the ``LtiConfiguration`` would
+be identical amongst the different XBlocks.
+
+In an LTI 1.3 launch, the ``location`` field is used to determine the ``https://purl.imsglobal.org/spec/lti/claim/roles``
+and ``https://purl.imsglobal.org/spec/lti/claim/context`` claims of the LTI 1.3 launch. In an LTI 1.3 launch, the
+``https://purl.imsglobal.org/spec/lti/claim/roles`` claim is required, and the
+``https://purl.imsglobal.org/spec/lti/claim/context`` claim is optional. This couples the LTI 1.3 launches to the XBlock
+through the ``location`` field.
+
+Furthermore, LTI 1.3 launches are also dependent on functions imported from edx-platform. These functions are used `to
+get the requesting user's role
+<https://github.com/openedx/xblock-lti-consumer/blob/master/lti_consumer/plugin/views.py#L182>`_, `to get the requesting
+user's external ID <https://github.com/openedx/xblock-lti-consumer/blob/master/lti_consumer/plugin/views.py#L183>`_, `to
+get the course key from the location field
+<https://github.com/openedx/xblock-lti-consumer/blob/master/lti_consumer/plugin/views.py#L181>`_, etc. This prevents LTI
+1.3 launches from running outside of the platform.
+
+Open edX would like to support the following LTI use cases:
```

Comment:
nice 👍 

I think this really clearly states why we are doing this even if the audience doesn't have a deep knowledge of LTI yet. A suggestion tho, maybe we lead with this earlier in this section before getting into technical details?